src/runtime/components/InputMenu.vue

```diff
@@ -95,6 +95,11 @@ export interface InputMenuProps<T extends MaybeArrayOfArrayItem<I>, I extends Ma
   items?: I
   /** Highlight the ring color like a focus state. */
   highlight?: boolean
+  /**
+   * Determines if custom user input that does not exist in options can be added.
+   * @defaultValue false
+   */
+  creatable?: boolean | 'always' | { placement?: 'top' | 'bottom', when?: 'empty' | 'always' }
```

Comment:
Maybe we should rename it to `create` in `v3` for consistency? What do you think?

Reply:
Sure! I have no opinion on the naming 🙂 though I feel `create` sounds more of an event (that the component does emit now; we can also call it `allowCreate`) - I may have missed other components that provide a similar prop, could you point me to what other component(s) allow creation of input?

Reply:
You're right `create` is an event name. We don't really have such behavior yet, but I was thinking:

Instead of adding a prop to do this, wouldn't it be possible to add an `item` with `type: 'create'`: https://github.com/nuxt/ui/blob/v3/src/runtime/components/SelectMenu.vue#L24

Reply:
That could be quite challenging for users, no? Handling the label, and making `items` array be more and more dynamic/reactive; eventually one will require a wrapper around it anyway.

Reply:
Yeah maybe you're right, `create-item` prop then? 😅

Reply:
Sure! Changed to `create-item` 😄